docs/quickstart.md

```diff
@@ -18,6 +18,8 @@ The template will deploy in your Azure subscription the Following resources:
 - [IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/)
 - [Azure Function](https://azure.microsoft.com/en-us/services/functions/)
 - [Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
+- [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) (when opted in to use Azure Monitor)
+- [Log Analytics](https://docs.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-overview) (when opted in to use Azure Monitor)
```

Comment:
nit: you could put this in an Optional section

Reply:
Since with c29006667d50cd427ab3ba4de00e010df6bf7341 this is only one component, I decided to not make a separate section for this.